argus/backend/service/results_service.py

```diff
@@ -197,6 +198,17 @@ class ResultsService:
     def __init__(self):
         self.cluster = ScyllaCluster.get()
 
+    @cache
+    def _ignored_results(self, test_id: UUID) -> list[str]:
```

Comment:
When is this cache gonna be cleared ?

When Argus restarts ?

Reply:
no, it's per request - will be needed when we merge PR with drawing limit lines, where best results are also used, to avoid double query db.